Source/Internal/API/APIEvent.swift

```diff
@@ -75,16 +75,16 @@ internal struct APIEvent {
         if let storeProductId = root["productId"] {
             payload["storeProductExternalId"] = storeProductId
         }
```

Comment:
We don't need to parse those 4 values they are used only in the old product architecture

Reply:
See the commit bc0929e